content/plugins/provide-plugin.md

```diff
@@ -38,3 +41,11 @@ new webpack.ProvidePlugin({
   'window.jQuery': 'jquery'
 })
 ```
+
+### Use Vue.js(ES Modules)
+
+```javascript
+new webpack.ProvidePlugin({
+  Vue: ['vue/dist/vue.esm.js', 'default'] // v2.2.6
```

Comment:
> What is v2.2.6 referring to?

`Vue.js` version which I use when I wrote the document.

> Referring to the 'default' export does not show the power of this feature since it is the default anyway?

If modules use ES2015 `export` syntax(like ES Modules), need to specify the `default` property.
Here's the sample.

```js
// webpack.config.js

new webpack.ProvidePlugin({
  esmodules_default: './esmodules.js',
  esmodules_default2: ['./esmodules.js', 'default'],
  esmodules_foo: ['./esmodules.js', 'foo'],
  commonjs_default: './commonjs.js',
  commonjs_foo: ['./commonjs.js', 'foo']
})
```

```js
// esmodules.js

export default function() { console.log('Hello from ES Modules of default') }
export { foo }

const foo = function() { console.log('Hello from ES Modules of foo') }
```

```js
// commonjs.js

bar = function() { console.log('Hello from CommonJS of default') }
bar.foo = function() { console.log('Hello from CommonJS of foo') }
module.exports = bar
```

```js
// main.js

esmodules_default() // Uncaught TypeError: esmodules_default is not a function
esmodules_default2() // Hello from ES Modules of default
esmodules_foo() // Hello from ES Modules of foo
commonjs_default() // Hello from CommonJS of default
commonjs_foo() // Hello from CommonJS of foo
```

Since this is a problem related to [babel](https://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/), should not it not be taken up?
If so, I'll remove the usage of loading ES Modules.
I'll add an example you mentioned in any case.